imagine/index.js

```diff
@@ -1,28 +1,25 @@
-addEventListener("fetch", (event) => {
-  event.respondWith(handleRequest(event.request));
-});
-
-function objectToBase64url(payload) {
-  return arrayBufferToBase64Url(
-    new TextEncoder().encode(JSON.stringify(payload))
-  );
-}
-
+/* globals JWKN ISS JWKE JWKD JWKP JWKQ JWKDP JWKDQ JWKQI BUCKET_NAME */
```

Comment:
This is very funky. Are these variables really existent in the environment of the cloudflare worker? If not below code seems to be buggy / not doing what it is supposed to do to me.